extensions/hypersonic/src/hooks/use-todos.ts

```diff
@@ -31,6 +32,8 @@ export function useTodos() {
 
   const getInitialData = async () => {
     try {
+      const localDatabase = await loadDatabase()
+      setNotionDbUrl(formatNotionUrl(localDatabase.databaseUrl))
```

Comment:
We need to check if local database exists, because at the first time it breaks the app.
 
```suggestion
      const localDatabase = await loadDatabase()
      if (localDatabase.databaseUrl) {
        setNotionDbUrl(formatNotionUrl(localDatabase.databaseUrl))
       }
```